index.js

```diff
@@ -79,6 +80,11 @@ const fetch = (keyword, language) => {
       'user-agent': 'https://github.com/rafaelrinaldi/mdn'
     }
   };
+
+  if (web) {
+    return open(url);
```

Comment:
@greg-js Ideally `fetch()` always return a `Promise`. This is optional and can be addressed by another PR :+1: